bundles/org.openhab.binding.wolfsmartset/src/main/java/org/openhab/binding/wolfsmartset/internal/dto/GetParameterValuesDTO.java

```diff
@@ -53,8 +55,8 @@ public void setValues(List<ValueDTO> values) {
         this.values = values;
     }
 
-    public Boolean getIsNewJobCreated() {
-        return isNewJobCreated;
+    public @NonNullByDefault Boolean getIsNewJobCreated() {
```

Comment:
A non nullable Boolean is for me simply a boolean. 
Why not returning type boolean?

Reply:
I'm not familiar with java ;) My knowledge is based on .NET / C# and there a bool / boolean / Boolean is the same and is never null. The issue here is, that the ``isNewJobCreated`` is null, so i thought Boolean can get null, and i added the Nullcheck here to prevent the NullPointerException. 

So is Type boolean not the same as Type Boolean in Java?

Reply:
No boolean is a primitive type can be true or false. 
Boolean is an object that can be null or having a value true or false.